Shield/MainPage.xaml.cs

```diff
@@ -80,51 +81,85 @@ public MainPage()
         {
             Instance = this;
 
+            appSettings = (AppSettings)App.Current.Resources["appSettings"];
+
             InitializeComponent();
             model = new MainViewModel();
             dispatcher = CoreWindow.GetForCurrentThread().Dispatcher;
 
-            appSettings = (AppSettings) App.Current.Resources["appSettings"];
-
             Initialize();
-            StatusBar.GetForCurrentView().BackgroundOpacity = 0.50;
+
+            if (Windows.Foundation.Metadata.ApiInformation.IsTypePresent("Windows.Phone.UI.ViewManagement.StatusBar"))
+            {
+                if (StatusBar.GetForCurrentView() != null)
+                {
+                    StatusBar.GetForCurrentView().BackgroundOpacity = 0.50;
```

Comment:
This is very-much on purpose / by-design. This is the new way to do Universal... instead of a project per destination, we now 'ask the environment' if something exists, and then effect a change. Will comment for clarity - but - it is basically "if I'm on a phone (i.e. if I have a statusbar), affect the opacity of the status bar"